.circleci/config.yml

```diff
@@ -30,35 +30,42 @@ jobs:
           # Commands listed here are from the CircleCI PostgreSQL config docs:
           # https://circleci.com/docs/2.0/postgres-config/#postgresql-circleci-configuration-example
           command: |
-            sudo apt-get install postgresql-client
-            echo ‘/usr/lib/postgresql/9.6/bin/:$PATH’ >> $BASH_ENV
-
-      - run:
-          name: Install Node.js
-          command: |
-            sudo apt-get update && curl -sL https://deb.nodesource.com/setup_6.x | sudo bash - && sudo apt-get install -y nodejs
+            sudo apt-get update -qq && sudo apt-get install -y build-essential postgresql-client
+            echo '/usr/lib/postgresql/9.6/bin/:$PATH' >> $BASH_ENV
 
       - restore_cache:
           keys:
-          - fec-api-dependencies-{{ checksum "requirements.txt" }}-{{ checksum "requirements-dev.txt" }}-{{ checksum "package.json" }}
+          - fec-api-dependencies-{{ checksum "requirements.txt" }}-{{ checksum "requirements-dev.txt" }}-{{ checksum "requirements-ci.txt" }}-{{ checksum "package.json" }}
```

Comment:
I don't think we want to get rid of these, install all of those [here](https://github.com/18F/openFEC/pull/2670/files#diff-1d37e48f9ceff6d8030570cd36286a61R62) so we will only want to use the saved version if there are no changes in any of those files

Reply:
Never mind!  This fixes that